packages/manageState/src/openrpc.json

```diff
@@ -0,0 +1,66 @@
+{
+  "openrpc": "1.2.4",
+  "info": {
+    "title": "Confirm Snap",
```

Comment:
Why do we have this openrpc.json file here? This seems to be for the "Confirm" snap, was this copied over or auto generated?

Reply:
oh my bad i thought i changed that. I did in fact use the confirm snap as a template
fixing now